pkg/generator/type_oneof.go

```diff
@@ -167,6 +183,35 @@ func (o *OneOfType) emitJSONMarshalFunc() generator.Statement {
 	)
 }
 
+func (o *OneOfType) emitToStringFunc(ctx *GeneratorContext) generator.Statement {
+	stmts := make([]generator.Statement, 0)
+	for i, alt := range o.AlternativeTypes {
+		returnStatement := generator.NewReturnStatement(`"null"`)
+
+		ts, ok := alt.(TypeWithStringConversion)
+		if ok {
+			res := fmt.Sprintf("*a.%s", ts.EmitToString(o.alternativeName(i), ctx))
+			returnStatement = generator.NewReturnStatement(res)
+		}
+
+		stmts = append(
+			stmts,
+			generator.NewIf(
+				fmt.Sprintf("a.%s != nil", o.alternativeName(i)),
+				returnStatement,
+			),
+		)
+	}
+	stmts = append(stmts, generator.NewReturnStatement(`"null"`))
+
+	return generator.NewFunc(
+		generator.NewFuncReceiver("a", fmt.Sprintf("%s", o.Names.StructName)),
```

Comment:
`fmt.Sprintf("%s", o.Names.StructName)` is just `o.Names.StructName`, isn't it?

```suggestion
		generator.NewFuncReceiver("a", o.Names.StructName),
```